### PR TITLE
add pre-installed apps to list

### PR DESCRIPTION
**Pull request by Muqtxdir**
Hi,

This PR will add the ubuntu pre-installed GNOME Apps to `open-pr-on-icon-changes.yaml`

related: #3033

### CODE REVIEW

**Comment by Muqtxdir** [comment]
cc @Jupi007

File: .github/workflows/open-pr-on-icons-changes.yaml [hunk 1]
```diff
@@ -40,6 +40,133 @@ jobs:
...
+            src: data/icons/scalable,
+            dest: icons/upstream/baobab-symbolics.list
+          },
+          {
+            project: gnome-characters symbolics,
+            repo: "https://gitlab.gnome.org/GNOME/gnome-characters.git",
+            upstreambranch: master,
+            localbranch: gnome-characters-symbolics,
+            projectdir: gnome-characters,
+            src: data/icons/hicolor,
```
Comment (Jupi007) [inline .github/workflows/open-pr-on-icons-changes.yaml:59]: Maybe target `data/icons/hicolor/scalable` here.

**Review by Jupi007 — APPROVED** [review]
LGFM 👍 Good job @Muqtxdir 

But the current script shows its limits. It needs to be adapted to filter images (it actually list other files like meson, ui...), and for GCC icons are mixed with many other files and its annoying. It could be also nice to also make the listing recursive.

Can you make these change in another PR before merging this one?

File: .github/workflows/open-pr-on-icons-changes.yaml (hunk 1 above)
Comment (Muqtxdir) [reply .github/workflows/open-pr-on-icons-changes.yaml]: ok, :+1:

**Comment by Muqtxdir** [comment]
@Jupi007 should I merge master into this branch now?

**Comment by Jupi007** [comment]
Yes :)

**Pull request merged**